packages/lib/survey/service.ts

```diff
@@ -111,7 +111,7 @@ export const getSurvey = async (surveyId: string): Promise<TSurvey | null> => {
         });
       } catch (error) {
         if (error instanceof Prisma.PrismaClientKnownRequestError) {
-          console.error(error);
+          // console.error(error);
```

Comment:
@ShubhamPalriwala you removed this because of the tests? We currently don't have a very good solution for the error logging but we have this to mainly debug customer error messages.
I don't think that no error logging solution is the best solution here. We can either mock the logger in the tests or find a better logging solution in prod (e.g. special logger with env variables that doesn't log in tests?)

Reply:
donezo